luxjs/grid/grid.js

```diff
@@ -127,14 +134,24 @@
                 };
 
                 scope.delete = function($event) {
-                    var modalTitle, modalContent,
+                    var modalTemplate,
                         first_field = gridOptions.columnDefs[0].field,
                         repr_field = scope.gridOptions.reprField || first_field,
                         icon = '<i class="fa fa-trash"></i>',
-                        modalTemplate = 'grid/modal.tpl.html',
-                        results = [],
                         success = false,
-                        subPath = scope.options.target.path || '';
+                        subPath = scope.options.target.path || '',
+                        //
+                        item,
+                        info = angular.element($document[0].createElement('p'))
+                                    .css('font-weight', 600),
+                        warning = angular.element($document[0].createElement('p'))
+                                    .css({'font-weight': 600, 'color': 'red'})
+                                    .html('DANGER - THIS CANNOT BE UNDONE'),
```

Comment:
We need to be able to configure this.
Can we use an angular template instead?
The message should be added to ``gridDefaults`` so that it can be overwritten.
In addition, I would like to see classes rather than hardcoded css:
```
<p class="text-danger cannot-undo">{{message}}</p>
```
For example, and style the ``cannot-undo`` class in the ``ui`` extension.